tools/deployment/osquery.example.conf

```diff
@@ -31,9 +31,7 @@
 
     // Comma-delimited list of table names to be enabled.
     // This allows osquery to be launched with certain tables only.
-    //"enable_tables": "foo_bar,time",
-
-    "utc": "true"
+    //"enable_tables": "foo_bar,time"
```

Comment:
I'm not sure this comma matters -- this comment probably takes precedent. I suspect we need to remove the comma on line 8. 

And, to be honest, I'd probably move the logger_plugin to the bottom, since it'll be less confusing.